src/main/scala-2.12/org/apache/james/gatling/jmap/draft/RetryAuthentication.scala

```diff
@@ -11,7 +11,9 @@ import io.gatling.http.response.Response
 object RetryAuthentication {
 
   def execWithRetryAuthentication(scenario: HttpRequestBuilder, checks: Seq[HttpCheck]): ChainBuilder =
-    exec(scenario.check(status.in(200, 401).saveAs("statusCode")).check(checkIfOk(checks): _*))
+    doIf(session => !session.attributes.contains("accessTokenHeader")) {JmapAuthentication.authentication()}
+      .exec(session => session.set("statusCode", 900))
```

Comment:
what this `900` means?

Reply:
Nothing, just specifying a random value here so that the check later don't complain upon timeouts that statusCode session variable is undefined...

Clearly it is a hack.

I had a quick time at it, and did not found better. However it did remove noise upon timeouts missing a status code in my runs... Which is nice!

Reply:
What about, at least, giving this magic number a name and add a comment about it?